pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -377,13 +379,9 @@ func lastAssignmentOutdated(wl *workload.Info, cq *cache.ClusterQueueSnapshot) b
 // FlavorAssignmentMode.
 func (a *FlavorAssigner) Assign(log logr.Logger, counts []int32) Assignment {
 	if a.wl.LastAssignment != nil && lastAssignmentOutdated(a.wl, a.cq) {
-		if logV := log.V(6); logV.Enabled() {
-			keysValues := []any{
-				"cq.AllocatableResourceGeneration", a.cq.AllocatableResourceGeneration,
-				"wl.LastAssignment.ClusterQueueGeneration", a.wl.LastAssignment.ClusterQueueGeneration,
-			}
-			logV.Info("Clearing Workload's last assignment because it was outdated", keysValues...)
-		}
+		log.V(5).Info("Clearing Workload's last assignment because it was outdated",
+			"cq.AllocatableResourceGeneration", a.cq.AllocatableResourceGeneration,
+			"wl.LastAssignment.ClusterQueueGeneration", a.wl.LastAssignment.ClusterQueueGeneration)
```

Comment:
Instead of code change, why not using `"--zap-log-level=6"?

Reply:
To clarify, this is not meant for merging. These are just temporary logs. If meant for merging, the entire thing would need to be cleaned up.

Set them to 5 for convenience so that only the image needed to be updated to a dev one. 

I've also added a lot of other logging at level 5.